corehq/apps/locations/forms.py

```diff
@@ -96,6 +96,9 @@ def __init__(self, location, bound_data=None, is_new=False, user=None,
 
         self.custom_data = self.get_custom_data(bound_data, is_new)
 
+        self.custom_data.form.helper.label_class = 'col-sm-3 col-md-4 col-lg-2'
+        self.custom_data.form.helper.field_class = 'col-sm-4 col-md-5 col-lg-3'
```

Comment:
I think there's something wrong with this part - when I view a location edit form locally with custom data - it doesn't show up as an editable field.  To reproduce, go to `/a/<domain>/settings/locations/fields/` and add a field (it doesn't matter what).  Then go to a location page `/a/<domain>settings/locations/<location_id>/` and observe that the "Additional Information" section at the bottom is broken.  Here's what it looks like in production: https://www.commcarehq.org/a/ethan-commtrack/settings/locations/c2b96890a06a3d4d5e0a024c42ceeef6/